Docs (2.6): Add `key` argument for @eq operator (#449)
nuwave/lighthouse: docs/2.6/api-reference/directives.md

  postsByCategory(category: String @eq): [Post] @hasMany
}
```

> If `category` field does not match with the database column, you can use the `key` argument.

Reviewer: ```suggestion
If the name of the argument does not match the database column,
pass the actual column name as the `key`.
```

```graphql
type User {

>   # this will place a equal operator for `cat` database column

Reviewer: ```suggestion
```

Reviewer: Let's keep the code GraphQL sections clear of redundant comments
